ui/workshop.css

```
@@ -0,0 +1,394 @@
+#page-workshop .fdrop:hover { border-color:rgba(var(--surface-rgb),.22); }
+#page-workshop .fdrop.open { border-color:rgba(var(--accent-rgb),.5); }
+#page-workshop .fdrop i { font-size:15px; color:var(--text-muted); }
+#page-workshop .fdrop .chev { font-size:14px; transition:transform .18s ease; }
+#page-workshop .fdrop.open .chev { transform:rotate(180deg); }
+#page-workshop .fdrop.hot i { color:var(--amber); }
+#page-workshop .filters .grow { flex:1; }
+#page-workshop .fmenu { position:absolute; top:calc(100% + 6px); left:0; z-index:40; min-width:100%; white-space:nowrap;
+          background:var(--glass-fill); border:1px solid var(--glass-border); border-radius:11px; padding:5px;
+          box-shadow:0 14px 44px rgba(var(--shadow-rgb),.5); display:none; }
+#page-workshop .fdrop.open .fmenu { display:block; animation:ws_fmenuIn .14s ease; }
+@keyframes ws_fmenuIn { from{ opacity:0; transform:translateY(-5px); } to{ opacity:1; transform:translateY(0); } }
```

Stylelint (17.14.0)

[error] Expected keyframe name "ws_fmenuIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)